app/main/hotkeys.js

```diff
@@ -44,16 +42,40 @@ const HOTKEYS = new Map([
   ['CmdOrCtrl+0', bw => () => bw.webContents.send('zoom-reset')],
 ]);
 
+const CUSTOM_HANDLED_HOTKEYS = new Map([
+  ['CmdOrCtrl+Left', bw => () => bw.webContents.send('go-back')],
+  ['CmdOrCtrl+Right', bw => () => bw.webContents.send('go-forward')],
+]);
+
+const handlerSet = new Set();
+
 export function bindBrowserWindowHotkeys(browserWindow) {
+  const handler = handleGuestActiveElementChange.bind(null, browserWindow);
+  handlerSet.add(handler);
+  ipcMain.on('guest-active-element', handler);
+
   HOTKEYS.forEach((handlerFactory, accelerator) => {
     shortcut.register(browserWindow, accelerator, handlerFactory(browserWindow));
   });
 }
 
 export function unbindBrowserWindowHotkeys(browserWindow) {
+  handlerSet.forEach(handler => ipcMain.removeListener('guest-active-element', handler));
   shortcut.unregisterAll(browserWindow);
 }
 
+function handleGuestActiveElementChange(browserWindow, event, { details }) {
+  if (!details || details.nodeName !== 'INPUT') {
+    CUSTOM_HANDLED_HOTKEYS.forEach((handlerFactory, accelerator) => {
+      shortcut.register(browserWindow, accelerator, handlerFactory(browserWindow));
```

Comment:
Rather than binding and unbinding the shortcuts (is there overhead here?), we could also wrap the handlers to ignore the call rather than unregistering the shortcuts. I think that'd clean up a lot of this handling

```
['CmdOrCtrl+Left', bw => requiresChromeFocus(() => bw.webContents.send('go-back')]),
```

```
const chromeHasFocus = true;
const requiresChromeFocus = (handler) => {
  if (chromeHasFocus) {
    handler();
  }
}
const handleGuestActiveElementChange = (bw, e, { details }) => {
  chromeHasFocus = !details || details.nodeName !== 'INPUT';
};
```